conceptnet5/vectors/evaluation/wordsim.py

```diff
@@ -351,6 +432,8 @@ def evaluate_raw(frame, subset='dev'):
     results.loc['ws353'] = ws_score
     results.loc['ws353-es'] = ws_es_score
     results.loc['ws353-ro'] = ws_ro_score
+    results.loc['semeval2017-monolingual'] = semeval_monolingual
```

Comment:
I think we're going to need shorter labels here, as these labels are going to take up valuable space in graph legends and in string representations of Pandas tables. How about "semeval17-2a" and "semeval17-2b"?